test/span_context.js

```diff
@@ -91,4 +91,23 @@ describe ('SpanContext should', () => {
         assert.isOk(bufferEqual(Utils.encodeInt64(0x5), context.parentId));
         assert.equal(context.flags, 0x1);
     });
+
+    it ('return null on malformed traces', () => {
+        // TODO(oibe) do we need these tests?  We are the ones creating the serialized
+        // version of these fields.  They should never be corrupt.  We follow this practice
+        // in a lot of places.  Also it makes the code slower.
+        /*
```

Comment:
yes, we do. Headers can be corrupted by http routing layers. We need to make sure that we do not parse malformed IDs, and that we don't throw parsing exceptions.